ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -39,12 +43,34 @@ const PostBreakingNewsLabel = ({
 const PostHeaderBanner = ({
   isBreakingNews,
   breakingNewsLabelText,
+  timestamp: curated,
 }: {
   isBreakingNews: boolean;
   breakingNewsLabelText?: string;
+  timestamp: string;
 }) => {
+  const { timezone, locale, altCalendar, service, script } =
+    useContext(ServiceContext);
+
+  const isRelative = isTenHoursAgo(new Date(curated).getTime());
+
   return (
     <div css={styles.postHeaderBanner}>
+      <TimeStampContainer
+        css={styles.timeStamp}
+        timestamp={curated}
+        dateTimeFormat="DD MMMM YYYY"
+        format="DD MMMM YYYY"
+        locale={locale}
+        timezone={timezone}
+        service={service}
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-expect-error
```

Comment:
Please add the reason why you're expecting this error
```suggestion
        // @ts-expect-error: insert reason here
```